examples/tasks/df-influxdb.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": 1,
+    "schedule": {
+        "type": "simple",
+        "interval": "1s"
+    },
+    "workflow": {
+        "collect": {
+            "metrics": {
+                "/intel/procfs/*": {}
+            },
+            "config": {},
+            "process": null,
+            "publish": [
+                {
+                    "plugin_name": "influxdb",
+                    "config": {
+                        "host": "172.17.0.20",
```

Comment:
yes, it should be removed - at this moment this collector cannot be used with influxdb (https://github.com/intelsdi-x/snap-plugin-collector-df/issues/28)